### DISCUSSION TITLE
define lending period

### DISCUSSION THREAD

**Discussion by mimosel** [leihs/leihs#1562] [Q&A / Community Support]
We are thinking about using leihs for our multimedia equipment lending service. A feature we would need is the possibility to define a lending period. 

Example: We would like to define that stuff can only be lended out for 14 days. Nice to have would be if we could define different lending periods for different categories of items.

Is that possible?

**Comment by neoheat** [comment]
dear @mimosel 
- each user determines the duration of the reservation up to the maximum reservation period.
- max. reservation period (field `maximum_reservation_time`) can be defined in the settings, but works globally for all device parks
- there is no possibility to define a timeout per category
